src/utils/context.ts

```diff
@@ -102,8 +104,8 @@ export class Context implements Partial<CowContext> {
     return this.#context.appDataHash ?? DefaultCowContext.appDataHash
   }
 
-  get isDevEnvironment(): boolean {
-    return this.#context.isDevEnvironment ?? DefaultCowContext.isDevEnvironment
+  get env(): Env {
+    return this.#context.env ?? DefaultCowContext.env
```

Comment:
why the coalescing? this means `' '` (empty string) is ok, do we want that? maybe also why the class is forcing you to type cast as `Env` above

would suggest `this.#context.env || DefaultCowContext.env`